cmd/manager/main.go

```diff
@@ -129,6 +130,11 @@ func main() {
 		os.Exit(1)
 	}
 
+	if err := routev1.Install(mgr.GetScheme()); err != nil {
```

Comment:
I'm curious as to what this line is doing. I noticed when I commented out the equivalent in cincinnati_controller_test.go, a unit test failed with 

```
    TestEnsurePolicyEngineRoute/EnsurePolicyEngineRoute: cincinnati_controller_test.go:380: no kind is registered for the type v1.Route in scheme "pkg/runtime/scheme.go:101"
```

Reply:
That is the same as AddToScheme (which adds this version to the scheme), but AddToScheme is now deprecated.